db/re/item_db.conf

```diff
@@ -125762,6 +125762,195 @@ item_db: (
 	EquipLv: 120
 	Script: <" bonus2 bSubEle,Ele_Holy,10+(getrefine()/2); ">
 },
+{
+	Id: 15093
+	AegisName: "Hero_Plate"
+	Name: "Hero Plate"
+	Type: "IT_ARMOR"
+	Buy: 10
+	Weight: 4000
+	Defense: 120
+	Slots: 1
+	Job: {
+		Crusader: true
+		Knight: true
+	}
+	Upper: ["ITEMUPPER_NORMAL", "ITEMUPPER_UPPER", "ITEMUPPER_BABY", "ITEMUPPER_THIRD", "ITEMUPPER_THIRDBABY"]
+	Loc: "EQP_ARMOR"
+	EquipLv: 160
+	Script: <"
+		bonus bMdef,5;
+		bonus bMaxHPrate,5;
+		bonus bMaxSPrate,5;
+		bonus bMatk,10;
+		bonus bBaseAtk,10;
+		bonus bHit,10;
+		bonus bFlee,10;
+		bonus2 bSkillAtk,"LG_BANISHINGPOINT",10;
+		bonus2 bSkillAtk,"RK_STORMBLAST",10;
+		.@r = (getrefine()+1)*3;
+		bonus5 bAutoSpell,"AL_HEAL",10,.@r,BF_SHORT,0;
+		bonus5 bAutoSpell,"WZ_STORMGUST",10,.@r,BF_SHORT,1;
+	">
+},
+{
+	Id: 15094
+	AegisName: "Hero_Magic_Coat"
+	Name: "Hero Magic Coat"
+	Type: "IT_ARMOR"
+	Buy: 10
+	Weight: 800
+	Defense: 50
+	Slots: 1
+	Job: {
+		Sage: true
+		Wizard: true
+	}
+	Loc: "EQP_ARMOR"
+	EquipLv: 160
+	Script: <"
+		bonus bMatk,10;
+		bonus bMaxHPrate,5;
+		bonus bMaxSPrate,5;
+		bonus bFlee,10;
+		bonus2 bSkillAtk,"WL_CRIMSONROCK",10;
+		bonus2 bSkillAtk,"SO_PSYCHIC_WAVE",10;
+		.@r = getrefine();
+		if (.@r) {
+		   bonus bFixedCastrate,21-.@r/2;
+		   bonus bNoCastCancel2, 1;
+		   bonus bMdef,.@r/2;
+		}
+	">
+},
+{
+	Id: 15095
+	AegisName: "Hero_Judgement_Shawl"
+	Name: "Hero Judgement Shawl"
+	Type: "IT_ARMOR"
+	Buy: 10
+	Weight: 500
+	Defense: 77
+	Slots: 1
+	Job: {
+		Acolyte: true
+		Monk: true
+		Priest: true
+	}
+	Loc: "EQP_ARMOR"
+	EquipLv: 160
+	Script: <"
+		bonus bMaxHPrate,5;
+		bonus bMaxSPrate,5;
+		bonus bMatk,10;
+		bonus bBaseAtk,10;
+		bonus bHit,10;
+		bonus bFlee,10;
+		bonus2 bSkillAtk,"AB_JUDEX",10;
+		bonus2 bSkillAtk,"AB_ADORAMUS",10;
+		bonus2 bSkillAtk,"SR_DRAGONCOMBO",10;
+		bonus2 bSkillAtk,"SR_SKYNETBLOW",10;
+		bonus2 bSkillAtk,"SR_EARTHSHAKER",10;
+		.@r = getrefine()+1;
+		bonus5 bAutoSpell,"PR_KYRIE",10,.@r*3,BF_SHORT,0;
+		bonus5 bAutoSpell,"PR_LEXAETERNA",1,.@r*6,BF_MAGIC,1;
+	">
+},
+{
+	Id: 15096
+	AegisName: "Hero_Trade_Mail"
+	Name: "Hero Trade Mail"
+	Type: "IT_ARMOR"
+	Buy: 10
+	Weight: 500
+	Defense: 100
+	Slots: 1
+	Job: {
+		Alchemist: true
+		Blacksmith: true
+		Merchant: true
+	}
+	Loc: "EQP_ARMOR"
+	EquipLv: 160
+	Script: <"
+		bonus bMaxHPrate,5;
+		bonus bMaxSPrate,5;
+		bonus bMatk,10;
+		bonus bBaseAtk,10;
+		.@r = getrefine();
+		if (readparam(bStr) >= 90)
+		   bonus bStr,.@r/2;
+		if (readparam(bInt) >= 90)
+		   bonus bInt,.@r/2;
+		if (readparam(bAgi) >= 90)
+		   bonus bAgi,.@r/2;
+		if (readparam(bVit) >= 90)
+		   bonus bVit,.@r/2;
+		if (readparam(bDex) >= 90)
+		   bonus bDex,.@r/2;
+		if (readparam(bLuk) >= 90)
+		   bonus bLuk,.@r/2;
```

Comment:
the indents inside the if's should be using tabs

Reply:
Done